php/class-post-type.php

```diff
@@ -675,4 +706,55 @@ public function hide_disabled_publishing_actions( $post ) {
 		</style>
 		<?php
 	}
+
+	/**
+	 * Get conflicts settings
+	 *
+	 * @param \WP_Post $post post to compare conflict values.
+	 *
+	 * @return array
+	 */
+	public function get_conflicts_setting( $post ) {
+		global $wpdb;
+		$post = get_post( $post );
+		if ( ! $post || $post instanceof \WP_Error ) {
+			return array();
+		}
+		$return = array();
+		$content = $this->get_post_content( $post );
+		if ( empty( $content ) || ! is_array( $content ) ) {
+			return $return;
+		}
+		$settings = array_keys( $content );
+		$query = $wpdb->prepare( "SELECT ID, post_name, post_status, post_content FROM $wpdb->posts WHERE post_type = %s AND post_status IN ( 'draft', 'pending', 'future' ) AND ID != %d AND ( 1 = 2 ", self::SLUG, $post->ID );
```

Comment:
By adding that we can directly append OR in `$query .= $wpdb->prepare( 'OR post_content LIKE %s', '%' . $wpdb->esc_like( wp_json_encode( $setting_id ) ) . '%' );`
However i have updated code using `implode` and removed `1 = 2`